lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -1496,81 +1449,86 @@ class LogProxyConsumer(ConsumerBase):
     """
 
     on = LogProxyEvents()
-    _stored = StoredState()
 
     def __init__(
         self,
         charm,
-        log_files: list = [],
-        relation_name: str = "log_proxy",
+        log_files: list = None,
+        relation_name: str = "log-proxy",
```

Comment:
Add and use "const" analogous to `DEFAULT_RELATION_NAME`?

Reply:
done